lib/core/public/cleanup-plugins.js

```diff
@@ -23,14 +23,18 @@ function cleanupPlugins(resolve, reject) {
     });
   });
 
-  axe.utils.toArray(document.querySelectorAll('frame, iframe')).forEach(function (frame) {
+  var flattenedTree = axe.utils.getFlattenedTree(document.body)

```

Comment:
Turns out there was a lint error in this line (missing ;). Very strange this didn't get picked up in circle.